pipeline/requirements.txt

```diff
@@ -5,4 +5,6 @@ python-dotenv
 requests
 psycopg2-binary
 flask
-boto3
\ No newline at end of file
+boto3
+pytest-cov
+mypy_boto3_ses
```

Comment:
My bad I didn't say earlier, but this isn't necessary - I'll add the line to install this in the README :)